service/controller/resource/tcnp/create_test.go

```diff
@@ -31,28 +30,20 @@ var update = flag.Bool("update", false, "update .golden CF template file")
 //
 func Test_Controller_Resource_TCNP_Template_Render(t *testing.T) {
 	testCases := []struct {
-		name             string
-		ctx              context.Context
-		cr               infrastructurev1alpha2.AWSMachineDeployment
-		encrypterBackend string
+		name string
+		ctx  context.Context
+		cr   infrastructurev1alpha2.AWSMachineDeployment
 	}{
 		{
-			name:             "case 0: basic test with encrypter backend KMS",
-			ctx:              unittest.DefaultContext(),
-			cr:               unittest.DefaultMachineDeployment(),
-			encrypterBackend: encrypter.KMSBackend,
-		},
-		{
-			name:             "case 1: basic test with encrypter backend Vault",
-			ctx:              unittest.DefaultContext(),
-			cr:               unittest.DefaultMachineDeployment(),
-			encrypterBackend: encrypter.VaultBackend,
+			name: "case 0: basic test with encrypter backend KMS",
```

Comment:
That is just a basic test now and has nothing to do with encrypters anymore, as this is also not configurable anymore. 

```suggestion
			name: "case 0: basic test",
```